00_hello_world/main.go

```diff
@@ -8,6 +8,21 @@ import (
 
 var out io.Writer = os.Stdout
 
+func fib(i int) int {
+	var a = 0
+	var b = 1
```

Comment:
These two lines could be written `a, b := 0, 1`.

Reply:
Done. Thanks.